frontend/src/app/commonComponents/MultiSelect/MultiSelectDropdown/MultiSelectDropdown.test.tsx

```diff
@@ -509,16 +509,9 @@ describe("MultiSelectDropdown component", () => {
         />
       );
 
-      fireEvent.click(screen.getByTestId("multi-select-input"));
-      await userEvent.hover(
-        screen.getByTestId("multi-select-option-Strawberries")
-      );
-      fireEvent.keyDown(
-        screen.getByTestId("multi-select-option-Strawberries"),
-        {
-          key: "ArrowDown",
-        }
-      );
+      await userEvent.click(screen.getByTestId("multi-select-toggle"));
+      // clicks down arrow 7 times total to get to last option and then ensure focus is not lost when clicking down arrow again
+      await userEvent.keyboard("{ArrowDown>7/}");
```

Comment:
⚠️ used the `user-event` supplied `keyboard` call rather than `fireEvent` to test the same thing